modules/administration/pages/ansible-setup-control-node.adoc

```diff
@@ -66,10 +65,10 @@ For executing remediations you need to install the SCAP security guide package o
 
 
 [[configure-ansible-inventory-files]]
-== Creating Ansible inventory files
+== Create Ansible inventory files
 
 Ansible Integration tools deploy a playbook as an inventory file.
-Therefore, you should create one inventory file for each distinct OS listed in _Table 1_.
+Therefore, you should create one inventory file for each operating system listed in _Table 1_.
```

Comment:
~~Therefore, you should~~ Create one inventory file for each operating system listed in _Table 1_.

Reply:
thanks, Joseph!